pkg/cmd/util/output/restore_describer.go

```diff
@@ -39,7 +40,15 @@ import (
 	"github.com/vmware-tanzu/velero/pkg/util/results"
 )
 
-func DescribeRestore(ctx context.Context, kbClient kbclient.Client, restore *velerov1api.Restore, podVolumeRestores []velerov1api.PodVolumeRestore, details bool, insecureSkipTLSVerify bool, caCertFile string) string {
+func DescribeRestore(
```

Comment:
Do we also need to add this to the structured describe https://github.com/vmware-tanzu/velero/blob/main/pkg/cmd/util/output/backup_structured_describer.go?

Reply:
By far, the structured describer is only for backup. Restore doesn't have it yet.